modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/cluster/ClusterReplCommand.java

```diff
@@ -34,7 +35,8 @@
                 ClusterInitReplCommand.class,
                 ClusterStatusReplCommand.class,
                 TopologyReplCommand.class,
-                ClusterUnitReplCommand.class
+                ClusterUnitReplCommand.class,
+                PartitionStatesReplCommand.class
```

Comment:
Is it correct that `PartitionStatesReplCommand` is defined both here and in `RecoveryReplCommand`?